packages/babel-helper-builder-react-jsx/src/index.js

```diff
@@ -14,9 +14,11 @@ export default function(opts) {
   const visitor = {};
 
   visitor.JSXNamespacedName = function(path) {
-    throw path.buildCodeFrameError(
-      "Namespace tags are not supported. ReactJSX is not XML.",
-    );
+    if (opts.throwIfNamespace) {
+      throw path.buildCodeFrameError(
+        "Namespace tags are not supported. ReactJSX is not XML.",
```

Comment:
By the way, the error message sounds a bit strange to me. What does `ReactJSX is not XML` mean? What do you think if we drop this part?

Reply:
Something the react team would of said? They are just saying the React's version of JSX looks like XML (which supports namespace tags) but it's not

Reply:
Yes, I would prefer something like that.

Reply:
I tried something like this, feel free to reword this. 👍